tutorials/basic-topics/sorting/selection-sort.md

```diff
@@ -11,4 +11,93 @@ keywords:
   - selection sort
 ---
 
-<TutorialAuthors names="@TBC"/>
+<TutorialAuthors names="@Bobliuuu"/>
+
+## Overview
+
+Selection sort is a commonly used comparison-based sorting algorithm, that is also very simple to implement. 
+Selection sort works on the premise that two subarrays are maintained: one which is **sorted**, and one which is **unsorted**. 
+Each step, one more element of the array gets sorted, until the entire array is sorted. 
+
+- **subarray**: A subsequence or slice of an array
+
+## Algorithm
+
+The concept of selection sort is that each time, we find the minimum element in the unsorted subarray, and we put it at the end of the sorted subarray. 
+Consider an array with n distinct elements. Looping n times, we
+- Find the minimum element in the unsorted subarray
+- Move that element to the end of the sorted subarray
+
+![1](https://he-s3.s3.amazonaws.com/media/uploads/2888f5b.png)
```

Comment:
what is the source of this image? Did you make it yourself? Besides, the image doesn't relate to the problem below.